src/main/java/org/shanerx/tradeshop/utils/Utils.java

```diff
@@ -52,13 +59,15 @@
  */
 public class Utils {
 
-	protected final PluginDescriptionFile pdf = Bukkit.getPluginManager().getPlugin("TradeShop").getDescription();
-	protected final String PREFIX = "&a[&eTradeShop&a] ";
+	protected TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
 
-	protected final TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
+	protected PluginDescriptionFile pdf = plugin.getDescription();
+	protected String PREFIX = "&a[&eTradeShop&a] ";
+	private UUID KOPUUID = UUID.fromString("daf79be7-bc1d-47d3-9896-f97b8d4cea7d");
+	private UUID LORIUUID = UUID.fromString("e296bc43-2972-4111-9843-48fc32302fd4");
```

Comment:
All uppercade fields here should be final

Reply:
I changed how they were being set when i was trying to get around the Gson error and forget to set them back, fixed